swagger.json

```diff
@@ -1324,7 +1324,7 @@
                     "format": "int32"
                 },
                 "plug_connected_boiler": {
-                    "type": "boolean"
+                    "type": "int32"
```

Comment:
@clinique @IOOOTAlan CI is in place. You can see the error at https://travis-ci.org/cbornet/netatmo-swagger-decl/builds/262997598#L590